packages/dx-react-grid-bootstrap3/src/plugins/toolbar.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Toolbar as ToolbarBase } from '@devexpress/dx-react-grid';
+import { Toolbar as Root } from '../templates/toolbar';
+
+const FlexibleSpace = () => (<div style={{ flex: '1 0' }} />);
```

Comment:
Why is the FlexibleSpace component inlined here?

It is also strange to use `flex-grow: 1` here. We should set grow, shrink, and base size to 0. And add `marginRight/marginLeft: auto`.